.github/workflows/platforms.yml

```diff
@@ -83,14 +83,16 @@ jobs:
                                     < /dev/null > /dev/null
                 apt-get -qq install cmake g++ build-essential ccache libasound2-dev libgtest-dev \
                                     libopusfile-dev libsdl2-dev python3-setuptools python3-pip zlib1g-dev \
+                                    libspeexdsp-dev \
                                     < /dev/null > /dev/null
                 pip3 install meson ninja
                 ;;
               fedora*)
                 dnf -q -y update
                 dnf -q -y install git which curl findutils procps-ng tar zstd screenfetch
                 dnf -q -y install cmake desktop-file-utils gcc gcc-c++ gmock-devel gtest-devel \
-                                  libappstream-glib libatomic meson opusfile-devel SDL2-devel zlib-devel
+                                  libappstream-glib libatomic meson opusfile-devel SDL2-devel zlib-devel \
+                                  libspeexdsp-devel
```

Comment:
I've looked up the correct names for each platform in the package repos to make sure they're correct (but as I said, I'm flying blind here).

Reply:
On Fedora the package is called `speexdsp-devel`; in the evening I'll try your branch to verify i builds ok.

Reply:
Thanks for that, I appreciate it.